docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,5 @@
+# Release 0.11.0
+## Bug Fixes
+
+- Specify the default values configmap in Kustomize file when creating helmchart artifacts with default values (#332)
+- Update command line parsing error (#329)
```

Comment:
@Skarlso copied from the auto-generated release notes, got a better message to put? This is a bit generic :smile:

Reply:
How about this:

- Return an understandable error in case of missing command line arguments